students/sangyeon/users/views.py

```diff
@@ -29,7 +30,7 @@ def post(self, request):
 
             name         = input_data["name"]
             email        = input_data["email"]
-            password     = input_data["password"]
+            hashed_password   = input_data["password"]
```

Comment:
변수의 네이밍을 `hashed_password`라고 하셨으면 그 의미에 맞게 작성해주세요!
```suggestion
            password        = input_data["password"]
            hashed_password = bcrypt.hashpw(password.encode('UTF-8'), bcrypt.gensalt()).decode("UTF-8")
```

```python
            User.objects.create(
                name = name,
                email = email,
                password = hashed_password,
                phone_number = phone_number
            )
```

Reply:
중간에 hashed_password 변수를 지정해서 넣었습니다!